sdk/python/feast/registry.py

```diff
@@ -65,9 +65,79 @@
     "": "LocalRegistryStore",
 }
 
+REGISTRY_OBJECT_TYPE_TO_STR = {
+    "entities": "entity",
+    "feature_views": "feature view",
+    "on_demand_feature_views": "on demand feature view",
+    "request_feature_views": "request feature view",
+    "feature_services": "feature service",
+}
+
+REGISTRY_OBJECT_TYPES = REGISTRY_OBJECT_TYPE_TO_STR.keys()
+
 logger = logging.getLogger(__name__)
 
 
+class RepoContents(NamedTuple):
```

Comment:
consider having a separate class for this? This isn't super core to the registry function I think

Also, you have a typo in the docstrings (Feas -> Feast)